nbextensions/bulkImportCell/cellTabs.js

```diff
@@ -28,41 +34,46 @@ define([
          *          , etc.
          *       }
          */
-        constructor(options) {
-            this.bus = options.bus;
-            this.ui = options.ui;
-            this.tabToggleAction = options.toggleAction;
-            this.controlBarTabs = options.tabs;
-        }
+        let bus = options.bus,
+            ui,
```

Comment:
should this be `ui = options.ui`?

Reply:
Nope, that sets up its own. I fixed that in the comments.